src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = {truck, bulldozer, excavator};
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+            System.out.println();
```

Comment:
The empty line printed inside the loop (System.out.println();) is not necessary and does not add any informative value to the output. It would be better to remove it unless there is a specific requirement to have a blank line between the outputs of different machines.